### ISSUE TITLE
Deadlock at loss.backward()

### ISSUE THREAD

**Issue by honggudrnjs** [speedinghzl/CCNet#54]
Hello, thank you for sharing your impressive work!
However, I got an issue on running your code.
Deadlock always occurs at loss.backward() of second iteration(batch index 1).
I think this is multiprocessing issue, isn't it?
My torch version is 0.4.1 and python version is 3.6.1
I hope this issue be resolved soon!

**Comment by speedinghzl** [comment]
The issue may occur when the machine is out of GPU memory.   You can reduce the input size and try again.
